### PR TITLE
Fix/geonames importer

### PR DESCRIPTION
**Pull request by voidxnull**

### CODE REVIEW

File: bin/geonames-sql.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env babel-node
 import fs from 'fs';
-
```
Comment (indeyets) [inline bin/geonames-sql.js]: this newline was to delimit core-modules from npm-modules. (no need to revert this for pull-request though. this is minor)

File: bin/geonames-sql.js [hunk 2]
```diff
@@ -13,98 +12,142 @@ import slug from 'slug';
...
+      .update(attributes)
+      .toString();
+  } else {
+    query = knex(table)
+      .insert(attributes)
+      .toString();
+  }
+
+  // Knex escapes double quotes that don't need to be escaped (e.g. when using JSON.stringify).
+  return query.replace(/\\\\"/g, '\\"') + ';';
```
Comment (indeyets) [inline bin/geonames-sql.js]: 1. is there a bug for this?
2. did you check knex 0.10? there was something in changelog about escaping

I just feel this is very fragile.

File: bin/geonames-sql.js [hunk 3]
```diff
@@ -13,98 +12,142 @@ import slug from 'slug';
...
+ * @param {Array} queries - array of sql queries
+ * @returns {Promise}
+ */
+function executeQueries(queries) {
+  return Promise.all(
+    chunk(queries, 1000).map(batch => {
+      return knex.raw(batch.join('\n'));
+    })
+  );
+}
```
Comment (indeyets) [inline bin/geonames-sql.js]: again: knex 0.10 has official support for this: https://github.com/tgriesser/knex/pull/1182

**Comment by voidxnull** [comment]
Knex 0.10 doesn't have support for this. I don't need to insert, I need to insert or update (upsert).

**Pull request merged**